.github/workflows/main.yml

```diff
@@ -43,5 +43,5 @@ jobs:
           context: .
           platforms: linux/arm64,linux/amd64
           push: true
-          tags: keytelematics/docker-dotnetcore-runtime-aws:latest
+          tags: keytelematics/docker-dotnetcore-runtime-aws:7.0
```

Comment:
this will need to be:

```yml
tags:
    - keytelematics/docker-dotnetcore-runtime-aws:latest
    - keytelematics/docker-dotnetcore-runtime-aws:7
    - keytelematics/docker-dotnetcore-runtime-aws:7.0
```